webapi/src/main/scala/org/knora/webapi/util/ConstructResponseUtilV2.scala

```diff
@@ -314,7 +320,7 @@ object ConstructResponseUtilV2 {
                                           requestingUser: UserADM)(implicit stringFormatter: StringFormatter): MainResourcesAndValueRdfData = {
 
         // An intermediate data structure containing RDF assertions about an entity and the user's permission on the entity.
-        case class RdfWithUserPermission(assertions: ConstructPredicateObjects, maybeUserPermission: Option[EntityPermission])
+        //case class RdfWithUserPermission(assertions: ConstructPredicateObjects, maybeUserPermission: Option[EntityPermission])
```

Comment:
It's better to delete code than to comment it out. It will still be there in the Git history if we ever need it again.